.prettierrc

```diff
@@ -0,0 +1,15 @@
+{
+  "printWidth": 100,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": true,
+  "quoteProps": "preserve",
+  "jsxSingleQuote": false,
+  "trailingComma": "es5",
+  "bracketSpacing": true,
+  "jsxBracketSameLine": true,
+  "arrowParens": "avoid",
+  "requirePragma": false,
+  "htmlWhitespaceSensitivity": "css"
+}
```

Comment:
Refer this for details of each of these rules: https://prettier.io/docs/en/options.html#html-whitespace-sensitivity

Reply:
prettierrc removed.